packages/now-cli/test/fixtures/unit/now-dev-next-rewrites/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "scripts": {
+    "build": "next build",
+    "dev": "next",
+    "now-build": "next build"
+  },
+  "dependencies": {
+    "next": "^8.0.0",
```

Comment:
Is this bug specific to Next 8? The last release of Next 8.x was over a year ago so we should probably use latest.

Reply:
I copy pasted the `now-dev-next` test and only made very minor modifications to it to verify my specific condition. since `rewrites` and `routes` cannot be in the same file I couldn't evolve the existing test but I'll update my test to match the `now->vercel` rename.